android/src/main/kotlin/network/mysterium/wireguard_dart/WireguardDartPlugin.kt

```diff
@@ -217,6 +223,10 @@ class WireguardDartPlugin: FlutterPlugin, MethodCallHandler ,ActivityAware,Plugi
         Log.e(TAG, "connect - Can't connect to tunnel: $e, ${Log.getStackTraceString(e)}")
         flutterError(result, e.message.toString())
       }
+      catch (e: Exception) {
+        Log.e(TAG, "connect -  $e, ${Log.getStackTraceString(e)}")
```

Comment:
1. This does not log stacktrace properly. You need to pass Throwable separately after the message.
2. Properly case the messages: should be sentence cased.